frontend/src/components/Comment/PostComment.js

```diff
@@ -1,15 +1,23 @@
-import React from "react";
+import React, { useState }  from 'react'
```

Comment:
The same comment as previously stated appies here. You could easily use `React.useState`!